sql/database.sql

```diff
@@ -8053,7 +8014,7 @@ DROP TABLE IF EXISTS `rule_filter`;
 CREATE TABLE `rule_filter` (
   `id` varchar(31) NOT NULL DEFAULT '' COMMENT 'Maps to the id column in the clinical_rules table',
   `include_flag` tinyint(1) NOT NULL default 0 COMMENT '0 is exclude and 1 is include',
-  `required_flag` tinyint(1) NOT NULL default 0 COMMENT '0 is required and 1 is optional',
+  `required_flag` tinyint(1) NOT NULL default 0 COMMENT '0 is optional and 1 is required',
```

Comment:
I think this is reverting one of Hank's fixes. Not a biggie since just a comment and most important that remains as is in master.